dbms/src/Functions/FunctionsJson.h

```diff
@@ -1514,4 +1514,398 @@ class FunctionJsonDepth : public IFunction
         }
     }
 };
+
+class FunctionJsonValidOthers : public IFunction
+{
+public:
+    static constexpr auto name = "json_valid_others";
+    static FunctionPtr create(const Context &) { return std::make_shared<FunctionJsonValidOthers>(); }
+
+    String getName() const override { return name; }
+
+    size_t getNumberOfArguments() const override { return 1; }
+
+    bool useDefaultImplementationForNulls() const override { return false; }
```

Comment:
```suggestion
    // Although it is stated in https://dev.mysql.com/doc/refman/5.7/en/json-attribute-functions.html#function_json-valid that returns NULL if the argument is NULL,
    // both MySQL and TiDB will directly return false instead of NULL.
    bool useDefaultImplementationForNulls() const override { return false; }
```